docs/operations.md

```diff
@@ -0,0 +1,140 @@
+# Operations
+
+This page describes the commands required for common operations.
+
+All subsequent sections assume that:
+- Commands are run from the repository root, unless otherwise indicated by a `cd` command.
+- An Ansible vault secret is configured.
+- The correct private key is available to Ansible.
+- Appropriate OpenStack credentials are available.
+- Any non-appliance controlled infrastructure is avaialble (e.g. networks, volumes, etc.).
+- `$ENV` is your current, activated environment, as defined by e.g. `environments/production/`.
+- `$SITE_ENV` is the base site-specific environment, as defined by e.g. `environments/mysite/`.
+- A string `some/path/to/file.yml:myvar` defines a path relative to the repository root and an Ansible variable in that file.
+- Configuration is generally common to all environments at a site, i.e. is made in `environments/$SITE_ENV` not `environments/$ENV`.
+
+Review any [site-specific documentation](site/README.md) for more details on the above.
+
+# Deploying a Cluster
+
+This follows the same process as defined in the main [README.md](../README.md) for the default configuration.
+
+Note that tags as defined in the various sub-playbooks defined in `ansible/` may be used to only run part of the tasks in `site.yml`.
+
+# SSH to Cluster Nodes
+This depends on how the cluster is accessed.
+
+The script `dev/ansible-ssh` may generally be used to connect to a host specified by a `inventory_hostname` using the same connection details as Ansible. If this does not work:
+- Instance IPs are normally defined in `ansible_host` variables in an inventory file `environments/$ENV/inventory/hosts{,.yml}`.
+- The ssh user is defined by `ansible_user`, default is `rocky`. This may be overriden in your environment.
+- If a jump host is required the user and address may be defined in the above inventory file.
+
+# Modifying general Slurm.conf parameters
+Parameters for [slurm.conf](https://slurm.schedmd.com/slurm.conf.html) can be added to an `openhpc_config_extra` mapping in `environments/$SITE_ENV/inventory/group_vars/all/openhpc.yml`.
```

Comment:
We seem to alternate between `$ENV` and `$SITE_ENV` in various places, is this intentional?
```suggestion
Parameters for [slurm.conf](https://slurm.schedmd.com/slurm.conf.html) can be added to an `openhpc_config_extra` mapping in `environments/$ENV/inventory/group_vars/all/openhpc.yml`.
```

Reply:
Yes, from first section on this page:

> - `$ENV` is your current, activated environment, as defined by e.g. `environments/production/`.
> - `$SITE_ENV` is the base site-specific environment, as defined by e.g. `environments/mysite/`.

generally we want people to put config in the site env.